### PR TITLE
ck3: validate courtier_guest_management

### PR DESCRIPTION
**Pull request by amtep**

### CODE REVIEW

File: src/ck3/data/courtier_guest_mgmt.rs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+#[derive(Clone, Debug)]
+pub struct CourtierGuestManagement {}
+
+inventory::submit! {
+    ItemLoader::Normal(GameFlags::Ck3, Item::CourtierGuestManagement, CourtierGuestManagement::add)
+}
+
+impl CourtierGuestManagement {
+    pub fn add(db: &mut Db, key: Token, block: Block) {
+        db.add(Item::NameList, key, block, Box::new(Self {}));
```
Comment (my4ng) [inline src/ck3/data/courtier_guest_mgmt.rs:25]: Shouldn't this be `Item::CourtierGuestManagement`?

File: src/ck3/data/courtier_guest_mgmt.rs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                    });
+                    vd.field_validated_key("desc", |key, bv, data| {
+                        validate_desc(bv, data, &mut sc_builder(key));
+                    });
+                });
+            });
+        } else {
+            let msg = "expected either `courtier_management` or `guest_management`";
+            err(ErrorKey::Validation).msg(msg).loc(key).push();
+            return;
```
Comment (my4ng) [inline src/ck3/data/courtier_guest_mgmt.rs:85]: don't see why we need the `return` there

**Review by my4ng — CHANGES_REQUESTED** [review]

File: src/ck3/data/courtier_guest_mgmt.rs (hunk 1 above)
Comment (amtep) [reply src/ck3/data/courtier_guest_mgmt.rs:25]: Oops yes

File: src/ck3/data/courtier_guest_mgmt.rs (hunk 2 above)
Comment (amtep) [reply src/ck3/data/courtier_guest_mgmt.rs:85]: Ah yeah that was left over after restructuring the code

**Pull request merged**